app/containers/message/Urls.tsx

```diff
@@ -48,21 +48,51 @@ const styles = StyleSheet.create({
 		height: 150,
 		borderTopLeftRadius: 4,
 		borderTopRightRadius: 4
+	},
+	imageWithoutContent: {
+		borderRadius: 4
+	},
+	activityPosition: {
+		zIndex: 2,
+		position: 'absolute'
+	},
+	loadingState: {
+		height: 0,
+		borderWidth: 0
 	}
 });
 
 const UrlImage = React.memo(
-	({ image }: { image: string }) => {
+	({
+		image,
+		setImageLoadedState,
+		style
+	}: {
+		image: string;
+		setImageLoadedState(value: TImageLoadedState): void;
+		style: StyleProp<ImageStyle>;
+	}) => {
 		const { baseUrl, user } = useContext(MessageContext);
 
 		if (!image) {
 			return null;
 		}
 
 		image = image.includes('http') ? image : `${baseUrl}/${image}?rc_uid=${user.id}&rc_token=${user.token}`;
-		return <FastImage source={{ uri: image }} style={styles.image} resizeMode={FastImage.resizeMode.cover} />;
+
+		return (
+			<>
+				<FastImage
+					source={{ uri: image }}
+					style={[styles.image, style]}
+					resizeMode={FastImage.resizeMode.cover}
+					onError={() => setImageLoadedState('error')}
+					onLoad={() => setImageLoadedState('done')}
+				/>
+			</>
+		);
 	},
-	(prevProps, nextProps) => prevProps.image === nextProps.image
```

Comment:
This test was unnecessary before this PR, since `image` is primitive and the only prop, so it would work without it.
If your PR is merged, there's no reason to use memo anymore, so you should remove it.